_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
+
+global.directory = 'file://' + __dirname + '/'
+
+const shadyServer = require('express')(),
+shadowServer = require('express')(),
```

Comment:
Not completely sure about server-side JS conventions, but I think it's cleaner if you `require('express')` once and reuse that:

```js
const express = require('express'),
  shady = express(),
  shadow = express();
```